Shared/mods/deathmatch/logic/Enums.cpp

```diff
@@ -161,4 +161,8 @@ ADD_ENUM1(PACKET_ID_WEAPON_BULLETSYNC)
 ADD_ENUM1(PACKET_ID_PED_TASK)
 ADD_ENUM1(PACKET_ID_PLAYER_NO_SOCKET)
 ADD_ENUM1(PACKET_ID_PLAYER_NETWORK_STATUS)
+ADD_ENUM1(PACKET_ID_PLAYER_ACINFO)

```

Comment:
These packet IDs weren't in enum info struct, I was not sure when I was adding the newly added missing ones including mine.